### PR TITLE
fix(zod-validator): validator support async hook

### PR DESCRIPTION
**Pull request by EdamAme-x**
https://github.com/orgs/honojs/discussions/2804#discussioncomment-9586541

### CODE REVIEW

**Comment by EdamAme-x** [comment]
I'll changeset later.

**Review by shishamo — APPROVED** [review]
LMGT!

**Comment by EdamAme-x** [comment]
@yusukebe I ready to review

**Review by yusukebe — COMMENTED** [review]
Please do not include any changes to this PR other than `packages/zod-validator`.

**Comment by EdamAme-x** [comment]
ready

**Comment by EdamAme-x** [comment]
I just ready

**Comment by yusukebe** [comment]
@EdamAme-x 

A changeset is not detected. Please remove it and re-create a changeset.

**Comment by EdamAme-x** [comment]
I am not familiar with changesets so I don't know if I can do it, but I will try.

File: packages/zod-validator/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # @hono/zod-validator
 
-## 0.2.1
```
Comment (yusukebe) [inline packages/zod-validator/CHANGELOG.md:3]: Please don't change this file.

Comment (yusukebe) [reply packages/zod-validator/CHANGELOG.md:3]: @EdamAme-x 

Can you check this?

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@EdamAme-x 

Thanks! Let's go with this.

**Pull request merged**